go.sum

```diff
@@ -94,6 +98,8 @@ github.com/IBM/go-sdk-core/v5 v5.2.0 h1:fVqzyAh9AiEOOcKZHVClmaH78qiYdxcUArP/27yo
 github.com/IBM/go-sdk-core/v5 v5.2.0/go.mod h1:vyNdbFujJtdTj9HbihtvKwwS3k/GKSKpOx9ZIQ6MWDY=
 github.com/IBM/go-sdk-core/v5 v5.3.0 h1:ZDqawFURQ1nlNhy4SE9Q+0AOlpa/TrsbVC7TnoSiTC8=
 github.com/IBM/go-sdk-core/v5 v5.3.0/go.mod h1:+MNa5Jbqb9FO7KEevo982Pb/YXr4adkyEffJlPs2TGc=
```

Comment:
I guess some other module still uses 5.3 indirectly. Since dep graph is handled by `go mod` automatically and we'll have to rerun it after rebase against upstream right before opening PR anyway, I'm holding on looking too closely on what's going on here.